cmd/algorand-indexer/main.go

```diff
@@ -23,12 +22,36 @@ import (
 	"github.com/algorand/indexer/version"
 )
 
+const autoLoadIndexerConfigName = config.FileName + "." + config.FileType
```

Comment:
`FileType` is `yaml`, it looks like `yml` is also valid so we should change it to match your documentation:
https://github.com/spf13/viper/blob/da55858fffe6093e995244bfc0aa2d9fba370e30/viper.go#L302